src/VideoNativeComponent.ts

```diff
@@ -3,13 +3,45 @@ import type {
   NativeSyntheticEvent,
   ViewProps,
 } from 'react-native';
+import type {
+  Drm,
+  DebugConfig,
+  Headers,
+  TextTracks,
+  SelectedTextTrack,
+  SelectedVideoTrack,
+  SubtitleStyle,
+  BufferConfig,
+  SelectedAudioTrack,
+} from './types/video';
 import {NativeModules, requireNativeComponent} from 'react-native';
+import type {
+  OnAudioFocusChangedData,
+  OnAudioTracksData,
+  OnBandwidthUpdateData,
+  OnVideoBufferData,
```

Comment:
There is no `OnVideoBufferData` type 🤔 did you mean `OnBufferData` ?

Reply:
good catch, I will fix it